server.go

```diff
@@ -698,10 +698,11 @@ func (sp *serverPeer) OnTx(_ *peer.Peer, msg *wire.MsgTx) {
 
 // OnBlock is invoked when a peer receives a block bitcoin message.  It
 // blocks until the bitcoin block has been fully processed.
-func (sp *serverPeer) OnBlock(_ *peer.Peer, msg *wire.MsgBlock, buf []byte) {
+func (sp *serverPeer) OnBlock(p *peer.Peer, msg *wire.MsgBlock, buf []byte) {
 	// Convert the raw MsgBlock to a bchutil.Block which provides some
 	// convenience methods and things such as hash caching.
 	block := bchutil.NewBlockFromBlockAndBytes(msg, buf)
+	peerLog.Infof("found block %d from %s", msg.Header.Version, p.Addr())
```

Comment:
NIT: `Found block` =)

Reply:
ups. removed it. Thats from another testing branch